main.cpp

```diff
@@ -20,12 +20,24 @@
 #if TARGET_UBLOX_EVK_ODIN_W2
 #include "OdinWiFiInterface.h"
 OdinWiFiInterface wifi;
+
+#elif TARGET_NUCLEO_L476RG
+#include "ESP8266Interface.h"
+// ESP8266_WIFI TX to be connected to D2 (Serial1_Rx)
+// ESP8266_WIFI_RX to be connected to D8 (Serial1_Tx)
+ESP8266Interface wifi(D8, D2);
+
+#elif  TARGET_NUCLEO_F429ZI
```

Comment:
@bulislaw can we add the pin config to the mbed_app.json, as it's done for the Client example?